agent/consul/internal_endpoint.go

```diff
@@ -144,6 +144,45 @@ func (m *Internal) ServiceDump(args *structs.ServiceDumpRequest, reply *structs.
 		})
 }
 
+func (m *Internal) ServiceTopology(args *structs.ServiceSpecificRequest, reply *structs.IndexedServiceTopology) error {
```

Comment:
I guess part of your TODO but a comment saying which API path this corresponds to would be ace. I came looking but i see it's not actually hooked up to one yet. Do you have a idea what the path will be?